keyboards/fjlabs/polaris/readme.md

```diff
@@ -0,0 +1,12 @@
+# Polaris-Compatible Hotswap and Solder PCB's
+
+The following is the QMK Firmware for the Polaris-Compatible Hotswap and Solder PCB's developed by FJLaboratories (FJLabs).
+* Keyboard Maintainer: FJLabs
+* Hardware Supported: Polaris Hotswap and Solder
+ 
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to jump to bootloader?

Reply:
Added